doc/Uplink.xml

```diff
@@ -359,10 +359,11 @@
           <para>Error : Optional user readable error information (readonly).</para>
         </listitem>
       </itemizedlist>
-      <para>The device shall interpret the field RemoteAddress as key to a specific uplink
-        configuration.</para>
-      <para>A device shall reject a configuration containing both CertificateID and
-        AuthorizationServer. </para>
+      <para>A device shall interpret the field RemoteAddress as key to a specific uplink
+        configuration and it shall support configuring the opional parameters CertificateID and
```

Comment:
typo correction: opional parameters-> optional parameters

Reply:
<para>AuthorizationServer :  JWTConfiguration token referring to an Authorization server
            that provides JWT token to authorize with the uplink server.</para>
        </listitem>==>this seems to be  authorization server configuration token. I do not see any token in JWTConfiguration.

Reply:
Agree, suggest below text instead "The AuthorizationServerConfiguration token refers to an authorization server that issues JWT tokens for authenticating with the uplink server." ,
Same change might be needed in **uplink wsdl** also.

Reply:
I guess I have already submitted such a change request in my earlier PRs
Ref: https://github.com/onvif/specs/pull/456?